ci/compare_sizes/project_instance.py

```diff
@@ -68,7 +59,9 @@ def _examples(self):
     def build_artifacts(self, cache):
         for example in self._examples:
             print(f"Building {example.name}...", file=sys.stderr)
-            self._build_artifact(example, cache)
+            for _, dirs, _ in os.walk(example.path):
+                if "src" in dirs:
+                    self._build_artifact(example, cache)
```

Comment:
this has obviously not been tested (as indicated by another comment with a run attempt). 

I suggest hardcoding all contract paths in a list, like `examples/cross-contract-calls/high-level`, 
`examples/cross-contract-calls/low-level` , `examples/mpc-contract/Cargo.toml`, 
and skipping dirs with (workspace roots, not actual contracts) virtual manifests , like `examples/cross-contract-calls`.